packages/kit/src/runtime/app/stores.js

```diff
@@ -55,3 +55,19 @@ export const updated = {
 		);
 	}
 };
+
+/**
+ * @template {keyof ReturnType<typeof getStores>} Name
+ * @param {Name} name
+ * @returns {ReturnType<typeof getStores>[Name]}
+ */
+function get_store(name) {
+	try {
+		return getStores()[name];
+	} catch (e) {
+		throw new Error(
+			`Store '${name}' is not available outside of a Svelte component on the server, as its bound to component context there. ` +
+				'For more information, see https://kit.svelte.dev/docs/state-management#careful-with-global-state'
```

Comment:
heads up to update the link if there is change based on this thread: https://github.com/sveltejs/kit/pull/8547/files#r1085676189